docs/logs/getting-started-aspnetcore/Program.cs

```diff
@@ -56,9 +56,9 @@
 
 public static partial class ApplicationLogs
 {
-    [LoggerMessage(EventId = 1, Level = LogLevel.Information, Message = "Starting the app...")]
+    [LoggerMessage(Level = LogLevel.Information, Message = "Starting the app...")]
```

Comment:
why EventId are removed?

Reply:
Starting with .NET 8, EventId will be automatically generated based on the hash of Function name, and this is considered the best practice. Check https://github.com/open-telemetry/opentelemetry-dotnet/pull/4821#discussion_r1316757504